sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/ChangeFeedContextClient.java

```diff
@@ -51,6 +51,11 @@ <T> Flux<FeedResponse<T>> createDocumentChangeFeedQuery(CosmosAsyncContainer col
                                                             CosmosChangeFeedRequestOptions requestOptions,
                                                             Class<T> klass);
 
+    <T> Flux<FeedResponse<T>> createDocumentChangeFeedQuery(CosmosAsyncContainer collectionLink,
```

Comment:
I kind feel we should remove the method without the isSplitHandlingDisabled, so kind making us to think about the split handling choices

Reply:
?? That would break Spark handling? In CFP we want to expose the split - In Spark we don't need it. IN normal query change feed we also don't need it - because the continuationState deals with it.

Reply:
nvm - looks like this would only be applicable in CFP - yes, there I think it makes sense.

Reply:
Discussed offline, will leave this alone to scope the change in this PR to just estimator